### ISSUE TITLE
Open in DIM Optimizer Fails Due to Adding Fragments as Mods

### ISSUE THREAD

**Issue by NoCodeLeftBehind** [Mijago/D2ArmorPicker#71]
Currently, D2ArmorPicker seems to treat Void/Stasis fragments as mods. DIM, at least in the beta branch, does not treat fragments as mods but as part of the customizable subclass options in loadout builder. When you choose a D2ArmorPicker build result to "Open in DIM Optimizer," DIM doesn't show the build armor and gives an error that nothing was found sure to inability to add mods. If in DIM you delete the fragments that are incorrectly appearing under the mods section of the loadout builder, the armor from the D2ArmorPicker will appear.

In short, D2ArmorPicker needs to send the selected fragments to DIM as subclass customizations, not mods.

**Comment by robojumper** [comment]
This used to be a DIM limitation -- the optimizer links only support loadout mods, not subclass mods -- but with DIM's full loadout shares, D2ArmorPicker should be able to send a full loadout to DIM.
